### ISSUE TITLE
Changes in text language cannot be indicated programmatically - Accessibility

### ISSUE THREAD

**Issue by brichwin** [h5p/h5p-multi-choice#77]
It appears that the question editor does not support marking up changes in text language.

Please see https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/ for an excellent explanation of the concern. This mainly affects non-visual users that depend on screen-reading software for access with text-to-speech.

The related WCAG 2.0/2.1 Success Criteria is SC 3.1.2 (Level AA): https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html 

See https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H58 (Using language attributes to identify changes in the human language) for more info.

This is especially important in the context of using H5P in language instruction eTexts where switching between two (or more) languages in the text is quite common. For example, the need to write: 

> Select the correct phrase that represents the plural form of “el libro” (the Spanish for "the books")

which would need to be coded as:

  `<div>Select the correct phrase that represents the plural form of <span lang="es">&quot;el libro&quot;</span> (the Spanish for "the books")</div>`

This assumes a parent container is indicating the default language of the text (i.e.: `<html lang="en">`).

Thanks!

**Comment by maitall** [comment]
I have the same problem using H5P in foreign language exercises. There are no answers or comments here... How did you solve the problem?

**Comment by otacke** [comment]
On H5P.com there's already a version of H5P core/CKEditor integration that allows defining `lang` attributes and I was told that this would eventually find its way into the regular version.

**Comment by maitall** [comment]
Thanks!
